File: warpaint.py
#!/usr/bin/env python3


def warpaint(s):
    s = s.strip()
    return s


def cli():
    import sys
    s = open(sys.argv[1], encoding='utf8').read()
    print(warpaint(s))

if __name__ == '__main__':
    cli()

File: warpaint.js
#!/usr/bin/env node
(function () {
    function warpaint(s) {
        s = s.trim()
        return s
    }

    function cli() {
        var fs = require('fs'),
            s = fs.readFileSync(process.argv[2], {encoding: 'utf8'})
        console.log(warpaint(s))
    }

    if (typeof require == 'function' && typeof module == 'object') {
        if (require.main === module) cli()
        else if (module.exports) module.exports = warpaint
    }
    else if (typeof define == 'function' && define.amd)
        define(function () { return warpaint })
    else if (typeof window == 'object') window.warpaint = warpaint
}())
